docs/release-notes/release-notes-0.1.0.md

```diff
@@ -1,5 +1,10 @@
 # About this release
-This release is intended to be compatible with the [aeternity epoch 0.16.0](https://github.com/aeternity/epoch/releases/tag/v0.16.0) release. It includes all major features in the state of the development at that point with a few differences - state channel P2P logic isn't implemented and smart contracts don't support operations native to the Sophia language (Solidity smart contracts are still supported).
+This release is intended to be compatible with the [aeternity epoch 0.16.0](https://github.com/aeternity/epoch/releases/tag/v0.16.0) release. It implements the aeternity protocol for that version, so any on-chain actions made by an epoch node should be able to be validated and processed by an elixir node and vice versa (this includes adding spend/oracle/naming system/smart contract/state channel transactions to the mempool, propagating those transactions and added blocks to other nodes). One protocol mismatch is that smart contracts don't support operations native to the Sophia language (Solidity smart contracts are still supported).
```

Comment:
"so any" the so is not needed.
"should be able to be" we are doing this, so don't be hypothetic.